lib/node_modules/@stdlib/stats/base/dists/cauchy/mode/manifest.json

```
@@ -0,0 +1,79 @@
+{
+  "options": {
+    "task": "build",
+    "wasm": false
+  },
+  "fields": [
+    {
+      "field": "src",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "include",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "libraries",
+      "resolve": false,
+      "relative": false
+    },
+    {
+      "field": "libpath",
+      "resolve": true,
+      "relative": false
+    }
+  ],
+  "confs": [
+    {
+      "task": "build",
+      "wasm": false,
+      "src": [
+        "./src/main.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/math/base/napi/binary",
+        "@stdlib/math/base/assert/is-nan",
+        "@stdlib/constants/float64/eps"
```

Don't need `eps` in the "build" configuration, as it is only used in examples and benchmarks and the actual implementation in `main.c` does not rely on this dep:

```suggestion
        "@stdlib/math/base/assert/is-nan"
```

---

@Planeshifter  without eps Cl tests are not passing as it is getting used in benchmark and examples file

---

@vivekmaurya001 Sorry, I meant to only remove it from the `"task": "build",` configuration as it is not used in the implementation. It is used and needed in the configs for `benchmark` and `examples`.

---

ook thanks for clarifying, im removing it from task build